### PR TITLE
Kelnera/alumni page changes

### PR DESCRIPTION
**Pull request by kelnera**

### CODE REVIEW

File: content/pages/organization/alumni.rst [hunk 1]
```diff
@@ -4,152 +4,43 @@ Alumni
...
-building web applications using Python, JavaScript and HTML/CSS. Outside of
-the OSL, he enjoys hacking on code and loves motorcycles - he owns a 2004
-Suzuki SV650S.
-
-.. raw:: html
-
-  </br></br>
+    :alt: Alex Polvi, co-founder and CEO, CoreOS
+
+*"I do not hestiate at all saying that my success is 100 percent because of the
```
Comment (leian7) [inline content/pages/organization/alumni.rst]: hesitate

File: content/pages/organization/alumni.rst [hunk 2]
```diff
@@ -4,152 +4,43 @@ Alumni
...
+OSL; they set me up to have a successful career. Without that, I don't know what
+I'd be doing. We need more technical people and more students who can go out and
+get jobs, and this is such a great model for that."*
+
+-Alex Polvi, co-founder and CEO, CoreOS
+
+Alex Polvi was one of the first student employees of the
+`Open Source Lab (OSL)`_, a CASS unit. He went on to found Cloudkick which was
+purchased by Rackspace. More recently, he and fellow OSL alumni, Brandon
+Philips, co-founded CoreOS. Read more about their story and how working at the
```
Comment (leian7) [inline content/pages/organization/alumni.rst]: "Read more": did you want a link here?

**Comment by Kennric** [comment]
@kelnera looks like this needs some rebasing now

**Comment by kelnera** [comment]
@osu-cass/cass-web Wow, so this is old. But is has a lot of changes that need to happen anyway. Could you guys check this out and let me know if there are any issues? I always get nervous before merging to master.

File: content/pages/organization/cass-students.rst [hunk 3]
```diff
@@ -52,7 +52,34 @@ libraries and tools for game developers.
...
+
+.. class:: no-breaks
+
+  **Github Account**
+
+https://github.com/ShannaChang
+
+.. raw::html
+
+  <br/>
```
Comment (leian7) [inline content/pages/organization/cass-students.rst]: Shanna and Taylor's entries are too close together:
close

**Comment by leian7** [comment]
Everything looks in order besides the spacing thing, and I'm assuming you're taking off some of these students in your other PR. I also checked in with kennric and thai about our workflow over the dev channel on irc today; you'll want to rebase this PR with staging, then merge this into staging so that we can eventually merge staging into develop. Let one of us know if you need help with this!

File: content/pages/organization/cass-students.rst (hunk 3 above)
Comment (kelnera) [inline content/pages/organization/cass-students.rst]: Could you try it again? The staging site and local host should not look like this. Justin Brown should be gone and Michael Elliott should be between Shanna and Taylor, which also solves the spacing issue.

Comment (leian7) [inline content/pages/organization/cass-students.rst]: Checked the staging site; I see what you mean. But when I try to build this branch locally, I still get this.

File: content/pages/organization/cass-students.rst [hunk 4]
```diff
@@ -52,7 +52,34 @@ libraries and tools for game developers.
...
+particularly enjoys working on front/end development. She comes from Taipei,
+Taiwan and in her spare time, she enjoys watching movies and running.
+
+.. class:: no-breaks
+
+  **Github Account**
+
+https://github.com/ShannaChang
+
+.. raw::html
```
Comment (leian7) [inline content/pages/organization/cass-students.rst]: The immediate fix for the spacing is to have a space between ``raw::`` and ``html``, i.e. ``.. raw:: html``.

Comment (kelnera) [inline content/pages/organization/cass-students.rst]: @leian7 Here's what I think is happening: I've had a lot of issues in the past of my desktop app picking up a ton of unnecessary commits and when I made this branch I was still using the desktop app. So these other commits are really outdated and aren't accurate anymore. Is this going to mess up what I just merged to master in my student additions pr?

**Pull request merged**